src/components/plotNFT/PlotNFTCard.tsx

```diff
@@ -320,13 +320,13 @@ export default function PlotNFTCard(props: Props) {
               </More>
             </Flex>
             <StyledInvisibleContainer>
-              <Typography variant="body2" noWrap>
+              <Typography component='div' variant="body2" noWrap>
                 {!!pool_url && (
                   <Flex alignItems="center" gap={1}>
                     <Typography variant="body2" color="textSecondary">
                       <Trans>Pool:</Trans>
                     </Typography>
-                    <Link target="_blank" href={pool_url}>
+                    <Link target="_blank" href={pool_url} to='/'>
```

Comment:
use href or to. not both

Reply:
let me fix that Thanks!

Reply:
to is used to not give an error that to is missing but let me fix that for the link module